src/sentry/migrations/0431_set_monitor_status_to_object_status_only.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 2.2.28 on 2023-04-28 23:49
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def set_monitor_status_to_object_status_only(apps, schema_editor):
+    Monitor = apps.get_model("sentry", "Monitor")
+
+    for checkin in RangeQuerySetWrapperWithProgressBar(
```

Comment:
If there are only 5k models it might be better to just skip `RangeQuerySetWrapperWithProgressBar`, that way we can use the index effectively.

Reply:
I'm all for it